remotes_env.sample

```diff
@@ -0,0 +1,10 @@
+export AWS_ACCESS_KEY_ID="...SET-ME..."
+export AWS_SECRET_ACCESS_KEY="...SET-ME..."
+export DVC_TEST_AWS_REPO_BUCKET="...SET-ME..."
+
+export GOOGLE_APPLICATION_CREDENTIALS=".gcp-creds.json"
+export GCP_CREDS="yes"
+export DVC_TEST_GCP_REPO_BUCKET="...SET-ME..."
+
+export AZURE_STORAGE_CONTAINER_NAME="dvc-test"
```

Comment:
let's move this file into `tests`?

Reply:
Ok

Reply:
I wonder if we should start with all "SET-ME" lines commented out to not try launching corresponding tests and let user uncomment only needed ones.

Reply:
@Suor , that sounds reasonable :+1: